readinformat.py

```diff
@@ -0,0 +1,61 @@
+'''Given files with data (in this example: ttbar, qcd, and wjets)
+   Concatenates all events
+   Creates Xjets, Xmuons, Xphotons, w, and y   
+
+'''
+
+
+import numpy as np
+from numpy.lib.recfunctions import stack_arrays
+from root_numpy import root2array
+import pandas as pd
+import glob
+import argparse
+
+def root2pandas(files_path, tree_name, **kwargs):
+	files = glob.glob(files_path)
+	ss = stack_arrays([root2array(fpath, tree_name, **kwargs).view(np.recarray) for fpath in files])
+	try:
+        	return pd.DataFrame(ss)
+    	except Exception:
+        	return pd.DataFrame(ss.data)
+
+def x(events, phrase):
```

Comment:
Please make the name of this function a bit more informative. In general, one letter names are reserved for ephemeral variables (indices, etc.).